index.html

```diff
@@ -88,14 +88,14 @@ <h4 class="cards-text-header bold font-size-32 line-height-44 black pad-t-8 slat
                         privately personalized reads; no accounts or sign-ups required.</p>
                     <ul class="d-flex pad-top-12 pad-bottom-2em">
                         <li
-                            class="font-size-12 font-weight-500 background-grey line-height-20 dodgerblue height-20 border-radius-7 width-53 pad-left-13">
+                            class="font-size-13 font-weight-500 background-grey line-height-20 dodgerblue height-20 border-radius-7 width-53 pad-left-13">
                             html</li>
                         <li
-                            class="font-size-12 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-45 pad-left-12">
+                            class="font-size-13 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-45 pad-left-12">
                             css
                         </li>
                         <li
-                            class="font-size-12 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-65 pad-left-7">
+                            class="font-size-13 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-65 pad-left-7">
                             javascript</li>
                     </ul>
```

Comment:
- Only two issues left to get approved:

- [ ] Kindly, work on Get my resume button to look more attractive, I suggest to do it as you did for `see project` button 
- [ ] Arrange JavaScript button well in the same line as you did for both HTML and CSS.

For more details:
![Selection_107](https://user-images.githubusercontent.com/55635977/126401765-6376ffe3-4df2-4afb-9845-b0c1f86b29ac.png)

Reply:
Done 👍🏽